microservices/app/orders/controllers.py

```diff
@@ -0,0 +1,54 @@
+import uuid
+
+from flasgger import swag_from
+from flask import Blueprint, jsonify, request
+
+from app.orders.apis import create_order
+from app.orders.models import Order
+
+mod_orders = Blueprint('orders', __name__, url_prefix='/orders')
```

Comment:
@Abezzam10 this is the sort of file we need for routes --- order routes should be organized in a controllers.py just like authentication, and registered in the flask app as well.